code/__DEFINES/mobs.dm

```diff
@@ -538,6 +537,13 @@ GLOBAL_LIST_INIT(xenoupgradetiers, list(XENO_UPGRADE_BASETYPE, XENO_UPGRADE_INVA
 #define SHRIKE_FLAG_PAIN_HUD_ON		(1<<0)
 #define SHRIKE_CURE_HEAL_MULTIPLIER	10
 
+//Drone defines
+
+#define DRONE_SALVAGE_BIOMASS_WINDUP		5 SECONDS //Delay before the target is salvaged
```

Comment:
Github is displaying an indentation issue with this line. please make sure all these defines are on the same indent.

Reply:
I think the Github display is fucked up; it looks fine on my end.